### PR TITLE
Map part1

### PR DESCRIPTION
**Pull request by borjanpopovic**
Added logic to. get your current position coordinates in the form by clicking on the "Get your position" button.
Alternatively you can look up the coordinates for the location you want to add in a tool like https://www.latlong.net/ and put in the coordinates manualy. 
The submitted location should be rendered on the map with the selected marker. 
If you click on the marker on the map a popup should open with the registered name, category and address.

Thanks for reviewing!

### CODE REVIEW

File: src/Components/LocationForm.js [hunk 1]
```diff
@@ -24,9 +24,28 @@ export default function LocationForm({ onAddLocations }) {
...
+
     setLocation({ ...location, [fieldName]: fieldValue });
   }
 
+  function lookupLocation(address) {
+    const apiUrl = `https://nominatim.openstreetmap.org/search?q=${address}&format=json`; // URLencode
+    fetch(apiUrl)
+      .then((result) => result.json())
+      .then((data) => {
+        // Rosenheimer 143, München
```
Comment (OliverGraml) [inline src/Components/LocationForm.js:39]: If you no longer need this comment, do not forget to delete it. :)

File: src/Components/LocationForm.js [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
+    latitude: '',
+    longitude: '',
+    address: '',
   };
 
   const [location, setLocation] = useState(initialLocation);
   const [isError, setIsError] = useState(false);
-  //console.log('added location', location);
+  /*  const [lat, setLat] = useState(null);
+  const [lng, setLng] = useState(null); */
```
Comment (OliverGraml) [inline src/Components/LocationForm.js]: Same here, do not forget to delete this.

**Review by OliverGraml — APPROVED** [review]
Looks very good to me. You found a good solution for your markers and popups. Keep it up!

File: src/Components/LocationForm.js (hunk 1 above)
Comment (borjanpopovic) [reply src/Components/LocationForm.js:39]: Alright, thank you :)

File: src/Components/LocationForm.js (hunk 2 above)
Comment (borjanpopovic) [reply src/Components/LocationForm.js]: Thank you, I'll delete it.

Comment (borjanpopovic) [reply src/Components/LocationForm.js]: Thanks a lot for your review!

**Pull request merged**